cypress/locators/toast/index.js

```diff
@@ -1,8 +1,7 @@
 import { TOAST_PREVIEW } from './locators';
 
 // component preview locators
-export const toastPreview = () => cy.iFrame(TOAST_PREVIEW).children();
-export const toastComponentIFrame = () => cy.iFrame(TOAST_PREVIEW);
-export const toastTogglePreview = e => cy.iFrame(`#${e}`);
-
+export const toastPreview = () => cy.get(TOAST_PREVIEW).children();
+export const toastTogglePreview = e => cy.get(`#${e}`);
```

Comment:
Move this `#${e}` as a separate locator please :)

Reply:
It's not a permanent locator, so it's not a good idea